speed-transition.lua

```diff
@@ -52,13 +53,21 @@ function speed_transition(_, sub)
          mark = speedup_zone_begin
          speedup_zone_end = mark+nextsub
          if shouldspeedup then
-            normalspeed = mp.get_property("speed")
-            if mp.get_property_native("video-sync") == "audio" then
-               mp.set_property("video-sync", "desync")
+            if skipmode and (nextsub-leadin>=leadin or nextsub==0) and mp.get_property("pause") == "no" then
```

Comment:
since we want to limit the range of leadin to be under lookahead and consequently nextsub (as we talked), the check `nextsub-leadin>=leadin` will always evaluate to true. This leaves `nextsub==0` for which you check in the next line.

Reply:
will change